Assets/Scripts/GameOverController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+
+public class GameOverController : MonoBehaviour {
+
+	public Text gameOverText;
+	private ScoreController scoreController;
+
+	// Use this for initialization
+	void Start () {
+		scoreController = GameObject.Find ("ScoreTracker").GetComponent<ScoreController>();
```

Comment:
We already have a GO called Controllers with the Game Controller and Input Controller. Should we also put the ScoreController on there?